project/Release.scala

```diff
@@ -40,18 +50,4 @@ object Release {
     commitNextVersion,
     pushChanges
   )
-
-  def publishEnabled = Seq(
-    publishArtifact := true, // Enable publish
-    publishMavenStyle := false,
-    // http://www.scala-sbt.org/0.12.2/docs/Detailed-Topics/Artifacts.html
-    publishArtifact in Test := false,
-    // Bintray
-    bintrayOrganization := Some("allenai"),
-    bintrayPackageLabels := Seq("scala"),
-    bintrayRepository := "private",
```

Comment:
This key was specified twice, and last writer wins, so the real fix was just deleting this key. I did a bit of re-org to make it less confusing.